[TW-1545] Admin overview refresh (#5309)
postmanlabs/postman-docs: src/pages/docs/administration/onboarding-checklist.md

## Set up your Postman team

> After you [set up Postman](#set-up-postman-in-your-organization), enable your teammates to find and join your team. You can also share invites with them directly. Once people are added to your team, you can begin assigning them roles based on the permissions each user needs in your team. Then you can begin collaborating with your teammates in workspaces, collections, and other workspace resources.

Reviewer: ```suggestion
After you [set up Postman](#set-up-postman-in-your-organization), enable your teammates to find and join your team. You can also share invites with them directly. Once you add members to your team, you can begin assigning them roles based on their permissions. Then you can begin collaborating with your teammates in workspaces, collections, and other workspace resources.
```
